src/assembly/local/quadrature_table.rs

```diff
@@ -266,6 +298,28 @@ where
     }
 }
 
+impl<T, GeometryDim, Data> UniformQuadratureTable<T, GeometryDim, Data>
+where
+    T: Scalar,
+    GeometryDim: DimName,
+    DefaultAllocator: Allocator<T, GeometryDim>,
+    Data: Clone,
```

Comment:
Maybe put `Data: Clone` above `DefaultAllocator` as by convention the allocator bounds in `fenris` come last